packages/sui-bundler/webpack.config.dev.js

```diff
@@ -94,6 +101,17 @@ let webpackConfig = {
       {
         test: /(\.css|\.scss)$/,
         use: ['style-loader', 'css-loader', 'postcss-loader', 'sass-loader']
+      },
+      {
```

Comment:
We should extract this to be reused across configs and only load if the config is set.